lang/cs/Org.Apache.REEF.Client/Avro/AvroBootstrapArgs.cs

```diff
@@ -0,0 +1,97 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using System.Runtime.Serialization;
+
+namespace Org.Apache.REEF.Client.Avro
+{
+    /// <summary>
+    /// Used to serialize and deserialize Avro record org.apache.reef.reef.bridge.client.AvroBootstrapArgs.
+    /// </summary>
+    [DataContract(Namespace = "org.apache.reef.reef.bridge.client")]
+    public sealed class AvroBootstrapArgs
```

Comment:
It's auto-generated with a few modifications, like all the other Avro files in .NET. I'll add a comment stating that.